.github/workflows/macos_building.yml

```diff
@@ -17,6 +17,11 @@ jobs:
     steps:
     - uses: actions/checkout@v2
 
+    - name: Install basics
+      run: |
+           brew update
+           brew install --cask homebrew/cask-versions/adoptopenjdk8
```

Comment:
we don't need install jdk8 and maven manully. please see this doc find more details : 
https://github.com/actions/virtual-environments/blob/main/images/macos/macos-11-Readme.md